Indicators/QuantConnect.Indicators.csproj

```diff
@@ -11,6 +11,8 @@
     <AssemblyName>QuantConnect.Indicators</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
+    <RestorePackages>true</RestorePackages>
```

Comment:
What is this restore packages flag for? Nuget packages should have already been set to restore on build. Was this required?